assets/admin/board_member_cpr_lookup.js

```diff
@@ -0,0 +1,36 @@
+/* global $ */
+
+$(function () {
+  const $cprLookupButton = $('.lookup-cpr')
+
+  $cprLookupButton.on('click', function () {
+    const $cprInputField = $('#BoardMember_cpr')
+    const $cpr = $cprInputField.val()
+
+    $.ajax({
+      url: '/case/new/apply-identifier-data',
+      type: 'POST',
+      data: {
+        type: 'CPR',
+        identifier: $cpr
+      },
+      success: function (response) {
+        if ($.isEmptyObject(response)) {
+          // No data returned, remove values for now
+          $('#BoardMember_name').val('')
+
+          // Indicate that identifier was not found
+          $($cprLookupButton).removeClass().addClass('btn-danger btn mt-2 lookup-cpr')
```

Comment:
I think you should just remove the `btn-primary` class and add `btn-danger`:

```js
….removeClass('btn-primary').addClass('btn-danger')
```

Reply:
It won't be sufficient to just remove `btn-primary` as it may have already been changed from previous attempts at finding information.

I guess we could detect which button style the button has and remove this specifically.